src/karma.config.js

```diff
@@ -69,14 +69,14 @@ module.exports = function (config) {
             // The existence of this preset option is due to a config issue with where jsx modules
             // are not transpiled to js.
             // See https://github.com/hypothesis/client/issues/2929
-            presets: require('../.babelrc.cjs').presets,
+            presets: require('../.babelrc-cjs.js').presets,
```

Comment:
Can you walk me through this a bit? The linked client issue https://github.com/hypothesis/client/issues/2929 is confusing me as it references a `preset-react` configuration property that doesn't appear to be set explicitly by our configuration. I am having trouble following.

Also, this elects to use the CJS configuration. Would it work with the ESM variant? Can you help me understand?

Reply:
On second review the common here is out of date. The tests *do* need to use the CJS configuration, but that's because the current bundler (Browserify) doesn't understand ES modules. When we replace the bundler we can remove this.